scenarios/kubernetes_build.py

```diff
@@ -142,7 +142,7 @@ def main(args):
         check('make', 'quick-release')
     else:
         check('make', 'release')
-    check(args.push_build_script, *push_build_args)
+    print(check_output(args.push_build_script, *push_build_args))
```

Comment:
```suggestion
    check_output(args.push_build_script, *push_build_args)
```

?
I'm not sure, tbh.

Reply:
@dims please bless us with your python knowledge :)

Reply:
@hasheddan -- given the usage elsewhere, I think just `check_output` should work.

Reply:
@justaugustus I see instances of this being set to a var, but I don't see any examples of this being called in a place where it is obvious that the returned value is printed